drivers/qemu/driver.go

```diff
@@ -346,6 +346,17 @@ func (d *Driver) StartTask(cfg *drivers.TaskConfig) (*drivers.TaskHandle, *drive
 		"-nographic",
 	}
 
+	var dnsArgs string
+	if cfg.DNS != nil {
+		if len(cfg.DNS.Servers) > 0 {
+			dnsArgs = ",dns=" + cfg.DNS.Servers[0]
+		}
+
+		for _, s := range cfg.DNS.Searches {
+			dnsArgs = fmt.Sprintf("%s,dnssearch=%s", dnsArgs, s)
+		}
+	}
```

Comment:
nit: would it be cleaner to keep a `[]string` of all things to be passed to `-netdev` `strings.Join`'d altogether?